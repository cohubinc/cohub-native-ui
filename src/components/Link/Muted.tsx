import React from "react";
import { TLinkProps } from "./TLinkProps";
import Base from "./Base";
import Typography from "../Typography";

export default ({ children, ...rest }: TLinkProps) => (
  <Base styled={false} {...rest}>
    <Typography muted>{children}</Typography>
  </Base>
);
