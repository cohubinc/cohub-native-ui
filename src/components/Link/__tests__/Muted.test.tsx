import React from "react";
import { shallow } from "enzyme";
import Muted from "..";

describe("Muted", () => {
  it("renders without crashing", async () => {
    const wrapper = shallow(<Muted to="/custom/path" />);
    expect(wrapper).toMatchSnapshot();
  });
});
