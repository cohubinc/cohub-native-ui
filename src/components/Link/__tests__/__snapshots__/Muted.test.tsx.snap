// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Muted renders without crashing 1`] = `
<Base
  animated={true}
  styled={true}
  to="/custom/path"
/>
`;
